### PR TITLE
Updated Rust code snippets to reflect version 0.12

### PR DESCRIPTION
**Pull request by sentinel1909**
I've taken a stab at updating the Rust code snippets in the original blog post, to reflect the version 0.12 changes.  The code in the associated GitHub repo still needs to be changed.  I figured the original author would probably want to work on that.

Cheers,

~Jeff Mitchell

### CODE REVIEW

File: _blog/2023-03-01-getting-started-with-rust-and-gpt.mdx [hunk 1]
```diff
@@ -296,8 +294,8 @@ This will allow our frontend to compile directly to the static folder that our b
 If we run `npm run build`, assuming there were no build errors we should now have a subfolder in our API directory called "static", which will hold all of our static assets that we can refer to in our Rust project. Let's implement our static folder by using the `shuttle_shared_folder` package:
 
 ```rust
-// lib.rs
-#[shuttle_service::main]
+// main.rs
+#[shuttle_runtime::main]
```
Comment (oddgrd) [inline _blog/2023-03-01-getting-started-with-rust-and-gpt.mdx:298]: Only thing missing is the return type of this function

**Review by oddgrd — APPROVED** [review]
Thanks a lot Jeff, this is great! :heart_eyes:  Only one tiny thing missing, I left a comment about it.

File: _blog/2023-03-01-getting-started-with-rust-and-gpt.mdx (hunk 1 above)
Comment (sentinel1909) [reply _blog/2023-03-01-getting-started-with-rust-and-gpt.mdx:298]: Oh man!  Will fix this evening and resubmit. :)

Comment (oddgrd) [reply _blog/2023-03-01-getting-started-with-rust-and-gpt.mdx:298]: Awesome, thanks! :pray: By the way, to fix it you can just create another commit on this branch, and then push that to `origin`, they all get added to this PR

**Comment by sentinel1909** [comment]
Ok...I think I got it and managed to update everything here.

**Comment by oddgrd** [comment]
Perfect, thanks!

**Pull request merged**